src/docfx/docset/Docset.cs

```diff
@@ -269,7 +269,12 @@ private JsonSchema LoadMetadataSchema(Config config)
                 var (_, content, _) = RestoreMap.GetRestoredFileContent(this, metadataSchema);
                 JsonUtility.Merge(token, JsonUtility.Parse(content, metadataSchema).value as JObject);
             }
-            return JsonUtility.ToObject<JsonSchema>(token).value;
+
+            var schema = JsonUtility.ToObject<JsonSchema>(token).value;
```

Comment:
seems now we are going to use json schema to validate reserved metadata, so why not apply to other metadata validation?😜